service/templates/projects.html

```diff
@@ -999,6 +999,85 @@ <h3>Virtual Workstations Commands</h3>
             </div>
           </div>
         </div>
+        <div id="plugin-info">
+          <div class="h3">Performance Plugin</div>
+          <div class="form-group">
+            <div class="col-lg-4 col-header">
+              <label class="control-label">Enabled&nbsp;&nbsp;
+                <input type="checkbox" name="perf_enabled" ng-model="settingsFormData.perf_enabled"/>
+              </label>
+              <div class="muted small">When checked, performance plugin will be enabled.</div>
+            </div>
+          </div>
+          <div class="h3">Build Baron Plugin</div>
+          <div class="form-group">
+            <div class="col-lg-4"> <label class="control-label"> Ticket create project </label> </div>
+          </div>
+          <div class="form-group">
+            <div class="col-lg-4">
+              <input class="form-control" type="text" ng-model="settingsFormData.build_baron_settings.ticket_create_project" placeholder="Ticket create project">
+            </div>
+          </div>
+          <div class="form-group">
+            <div class="col-lg-4"> <label class="control-label"> Ticket search project </label> </div>
```

Comment:
Now that we're showing this somewhere where non-admins can see it and won't understand it, I think maybe it's time to change this to something more helpful 😅  maybe "Project to Create Tickets In" / "Projects to Search"? Should also use proper capitalization for headings